calico-enterprise_versioned_docs/version-3.17/releases.json

```diff
@@ -1,4 +1,271 @@
 [
+  {
+    "title": "v3.17.4",
+    "tigera-operator": {
+      "image": "tigera/operator",
+      "version": "v1.30.8",
```

Comment:
Likely to be v1.30.10, as 8&9 already exist: https://github.com/tigera/operator/releases

Reply:
Done.